lib/java/desktop-environment/src/test/java/org/enso/desktopenvironment/DirectoriesTest.java

```diff
@@ -2,12 +2,13 @@
 
 import java.io.IOException;
 import java.nio.file.Files;
+import org.enso.common.Platform;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class DirectoriesTest {
```

Comment:
- shouldn't this test also run in _native image mode_?
- there doesn't seem to be the _NI testing_ infrastructure in `desktop-environment` project
- what about merging `desktop-environment` and `change-directory` into single `os-environment` project?
- then all the tests could run in _NI mode_

Reply:
Merged to single project in 59f7328965af29aa66a5e386de7476411cf5edb1

Reply:
Ensured that all these tests run in NI mode:
- 8c8cc2b5931663e01a312ea84cc8f7eadfc9cd63
- c6d04942de6295607dd12afa59ed74d04fd154e3